x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/rule_request_schema.mock.ts

```diff
@@ -213,3 +214,14 @@ export const getUpdateNewTermsSchemaMock = (
   new_terms_fields: ['user.name'],
   history_window_start: 'now-7d',
 });
+
+export const getCreateEqlRuleSchemaMock = (ruleId = 'rule-1'): EqlRuleCreateProps => ({
+  description: 'Event correlation index pattern rule',
+  name: 'Event correlation index pattern rule',
+  severity: 'high',
+  risk_score: 55,
+  rule_id: ruleId,
+  type: 'eql',
+  language: 'eql',
+  query: 'process where process.name == "regsvr32.exe"',
```

Comment:
Nit: It's a bit confusing that it's called "index pattern rule" but no index patterns are actually specified.

Reply:
188a053e010